src/constants.js

```diff
@@ -0,0 +1,14 @@
+// ⬇ these are typically the regions we have collectors in
+// we also support us-east-1 but due to DNS constraints we cannot list it here
+const SUPPORTED_REGIONS = new Map(
```

Comment:
A comment on why this is a `Map` versus a `Set`, please? (so you can add regional support for the signer API)

Reply:
When I ran this, Set had quite a bit slower perf:

Reply:
I thought it was also so you could add `us-east-1` in the case of signer

Reply:
Comment added